src/electionguard_gui/components/create_key_ceremony_component.py

```diff
@@ -44,8 +47,8 @@ def create_key_ceremony(
         )
         if existing_key_ceremonies:
             self.log.debug(f"record '{key_ceremony_name}' already exists")
-            result: dict[str, Any] = eel_fail("Key ceremony name already exists")
-            return result
+            fail_result: dict[str, Any] = eel_fail("Key ceremony name already exists")
```

Comment:
`mypy` is occasionally unable to determine types when you return a function call directly.  This is a bit annoying, but this seems to fix it.